File: sample/price.php
<?php
function urlsafe_b64decode($string)
{
  $data = str_replace(array('-','_','.'),array('+','/','='),$string);
  $mod4 = strlen($data) % 4;
  if ($mod4) {
    $data .= substr('====', $mod4);
  }
  return base64_decode($data);
}


$encryption_encoded_key = "sIxwz7yw62yrfoLGt12lIHKuYrK_S5kLuApI2BQe7Ac";
$integrity_encoded_key  = "v3fsVcMBMMHYzRhi7SpM0sdqwzvAxM6KPTu9OtVod5I";
$long_ciphertext = "SjpvRwAB4kB7jEpgW5IA8p73ew9ic6VZpFsPnA";

$long_ciphertext =  urlsafe_b64decode($long_ciphertext);
var_dump( strlen( $long_ciphertext ) );
require_once __DIR__ . DIRECTORY_SEPARATOR . ".." . DIRECTORY_SEPARATOR . "lib" . DIRECTORY_SEPARATOR . "DecryptPrice.php";
date_default_timezone_set( "Asia/Tokyo");
$DecryptPrice = new DecryptPrice( $encryption_encoded_key, $integrity_encoded_key );
var_dump( $DecryptPrice-> decrypt( $long_ciphertext ) );

File: python/sample/price.py
# import system files
import sys,os
import binascii
import base64

# get requirement files
dirname   = os.path.abspath(os.path.dirname(__file__)) 
dirsep    = os.sep
sys.path.append(  os.sep.join( [ os.path.dirname(os.path.abspath(__file__)) , '..', 'lib' ] ) )
from DecryptPrice import DecryptPrice

def _urlsafe_b64decode( s ):
    return base64.urlsafe_b64decode(s + '=' * (4 - len(s) % 4))

encryption_encoded_key = "sIxwz7yw62yrfoLGt12lIHKuYrK_S5kLuApI2BQe7Ac";
integrity_encoded_key  = "v3fsVcMBMMHYzRhi7SpM0sdqwzvAxM6KPTu9OtVod5I";
long_ciphertext = "SjpvRwAB4kB7jEpgW5IA8p73ew9ic6VZpFsPnA";

long_ciphertext =  _urlsafe_b64decode(long_ciphertext);

decrypt_price = DecryptPrice( encryption_encoded_key, integrity_encoded_key );
print decrypt_price.decryption( long_ciphertext );
